lib/passwordless/controller_helpers.rb

```diff
@@ -3,6 +3,12 @@
 module Passwordless
   # Helpers to work with Passwordless sessions from controllers
   module ControllerHelpers
+    # Returns the {Passwordless::Session} (if set) from the session.
+    # @return [Session, nil]
+    def current_passwordless_session(authenticatable_class)
+      @current_passwordless_session ||= Passwordless::Session.find_by(id: session[session_key(authenticatable_class)])
```

Comment:
I don't really like to memoize on behalf of the user. My guess is it is only going to lead to weird, hard to figure out inconsistencies? _Rose memoization_ is widely enough known that we can assume people know to do it if they want it.

The readme already suggests to memoize `current_user` which is, probably, mostly what these methods will be used for.